src/transformers/models/llava_next/image_processing_llava_next.py

```diff
@@ -603,6 +717,9 @@ def preprocess(
             pixel_values = np.array(pixel_values)
             new_images.append(pixel_values)
 
-        data = {"pixel_values": new_images, "image_sizes": image_sizes}
+        if do_pad:
```

Comment:
I override the `pad` from transforms and added the `do_pad` flag, but now we would need to add the `do_pad` to the `processing_config` in the hub for the tests to pass

@amyeroberts will this work? If yes I will ask @NielsRogge to update the processor config :)